src/imu/imu.c

```diff
@@ -51,10 +51,17 @@ static void init_filters(LoggerConfig *loggerConfig)
 void imu_sample_all()
 {
         for (size_t i = 0; i < CONFIG_IMU_CHANNELS; i++) {
-                update_filter(&g_imu_filter[i], imu_read(i));
+                update_filter(&g_imu_filter[i], imu_read_raw(i));
         }
 }
 
+float imu_read_cal_value(enum imu_channel channel )
```

Comment:
Looks like an unrelated change, please submit as part of a different PR when that feature is ready

Reply:
I knew you'd notice.  I struggle so much with git to pull things apart after the fact, I was hoping I could slide this in since it touches dependant files and is pretty innocuous, rather than tease it out.   If you really want me to relocate it I will.

Reply:
Please do. It seems pedantic but it really helps down the line to keep the concepts separate. Thanks!